lib/std/Build/Step/Run.zig

```diff
@@ -19,10 +20,10 @@ step: Step,
 /// See also addArg and addArgs to modifying this directly
 argv: ArrayList(Arg),
 
-/// Set this to modify the current working directory
+/// Use `setCwd` to set the initial current working directory
 /// TODO change this to a Build.Cache.Directory to better integrate with
 /// future child process cwd API.
```

Comment:
Please remove this TODO comment because it is misleading. I will open an issue for it instead if I want to pursue this idea.

```suggestion
```